sdk/textanalytics/azure-ai-textanalytics/pom.xml

```diff
@@ -13,7 +13,7 @@
 
   <groupId>com.azure</groupId>
   <artifactId>azure-ai-textanalytics</artifactId>
-  <version>1.0.0</version> <!-- {x-version-update;com.azure:azure-ai-textanalytics;current} -->
+  <version>5.0.0</version> <!-- {x-version-update;com.azure:azure-ai-textanalytics;current} -->
 
   <name>Microsoft Azure client library for Text Analytics</name>
   <description>This package contains the Microsoft Azure Cognitive Services Text Analytics SDK.</description>
```

Comment:
Should core and identity versions match the versions that 1.0.1 text analytics went out?